<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>961</width>
     <height>871</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout_10">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="bars1">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="bars2">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="bars3">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="bars4">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="bars5">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QLabel" name="bars6">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="bars7">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="bars8">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="bars9">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="bars10">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <widget class="QLabel" name="bars11">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="bars12">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="bars13">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="bars14">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="bars15">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <widget class="QLabel" name="bars16">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="bars17">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="bars18">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="bars19">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="bars20">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QGridLayout" name="gridLayout_9">
      <item row="0" column="0">
       <widget class="QLabel" name="bars21">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="bars22">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="bars23">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="bars24">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="bars25">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <layout class="QGridLayout" name="gridLayout_13">
      <item row="0" column="0">
       <widget class="QLabel" name="bars26">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="bars27">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="bars28">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="bars29">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="bars30">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="6" column="0">
     <layout class="QGridLayout" name="gridLayout_14">
      <item row="0" column="0">
       <widget class="QLabel" name="bars31">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="bars32">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="bars33">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="bars34">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="bars35">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="7" column="0">
     <layout class="QGridLayout" name="gridLayout_15">
      <item row="0" column="0">
       <widget class="QLabel" name="bars36">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="bars37">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="bars38">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="bars39">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="bars40">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="8" column="0">
     <layout class="QGridLayout" name="gridLayout_16">
      <item row="0" column="0">
       <widget class="QLabel" name="bars41">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="bars42">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="bars43">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="bars44">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="bars45">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="9" column="0">
     <layout class="QGridLayout" name="gridLayout_17">
      <item row="0" column="0">
       <widget class="QLabel" name="bars46">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="bars47">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="bars48">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="bars49">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="bars50">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../images/barexample.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>961</width>
     <height>170</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox#groupBox{background-color: rgb(89, 175, 255);
color: rgb(255, 255, 255);}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0" colspan="2">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: transparent;
color:white;</string>
          </property>
          <property name="text">
           <string>Product.     </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="searchp">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="placeholderText">
           <string> Search By   Product id or name or Barcode Here</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: transparent;
color:white;</string>
            </property>
            <property name="text">
             <string>Pro. Name </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="pname">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: transparent;
color:white;</string>
            </property>
            <property name="text">
             <string>Unit Name         </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="unit">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: transparent;
color:white;</string>
            </property>
            <property name="text">
             <string>Category   </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="category">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: transparent;
color:white;</string>
            </property>
            <property name="text">
             <string>Barcode Number</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="barcode">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>687</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <layout class="QGridLayout" name="gridLayout_11">
      <item row="0" column="0">
       <widget class="QPushButton" name="printb">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{padding:5px;
background-color: rgb(0, 0, 0);
border-radius:3px;
color:white;}

QPushButton:hover:!pressed {
    background-color:  #54d777;
    border-radius: 3px;
}</string>
        </property>
        <property name="text">
         <string>    Print   </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="showb">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{padding:5px;
background-color: rgb(0, 0, 0);
border-radius:3px;
color:white;}

QPushButton:hover:!pressed {
    background-color:  #54d777;
    border-radius: 3px;
}</string>
        </property>
        <property name="text">
         <string>Export Barcode Image</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
